ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: ubi
     namespace: ocp
     tag: "9"
+  tests-private:
```

Comment:
Was this included in this PR on purpose?

Reply:
Yes, this image is required by the [openshift-extended-tests](https://github.com/openshift/release/blob/bd1d761d67bd097f4cf48f802f7316abbde1b134/ci-operator/step-registry/openshift-extended/test/openshift-extended-test-ref.yaml#L3C1-L3C1), which is part of the the [openshift-e2e-test-qe-chain](https://github.com/openshift/release/blob/bd1d761d67bd097f4cf48f802f7316abbde1b134/ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml#L8).